tests/helpers.py

```diff
@@ -66,6 +66,10 @@ def setup_method(self, method):
         self.client = self.app.test_client()
         self.get_user_patch = None
 
+        self.buyer_dashboard_url = "/buyers"
```

Comment:
Thanks.. this variable should be named buyer_requirements_dashboard_url I think based on what it's actually containing?

Reply:
sure I'll get that updated, `briefs_dashboard_url` sounds more consistent.